### PR TITLE
Add bash (sh) header

### PR DESCRIPTION
**Pull request by TBouder**

### CODE REVIEW

File: lib/headers/Sh.header [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# **************************************************************************** #
```
Comment (Kerosene2000) [inline lib/headers/Sh.header]: Why not using the `Makefile.header` file ?

**Comment by TBouder** [comment]
Yes, it would avoid two identical files, you're right !

**Comment by Kerosene2000** [comment]
Thanks for your PR 😄

**Comment by zurbo** [comment]
Thx :')

**Pull request merged**